modules/openapi-generator/src/main/resources/java-micronaut/common/configuration/gradle/build.gradle.mustache

```diff
@@ -51,27 +64,33 @@ dependencies {
     implementation("io.swagger:swagger-annotations:1.6.5")
     {{/generateSwagger1Annotations}}
     {{#generateSwagger2Annotations}}
-    implementation("io.swagger.core.v3:swagger-annotations:2.2.0")
+    implementation("io.swagger.core.v3:swagger-annotations:2.2.19")
     {{/generateSwagger2Annotations}}
     runtimeOnly("ch.qos.logback:logback-classic")
+    {{#isMicronaut4OrGreater}}
+    runtimeOnly("org.yaml:snakeyaml")
+    {{/isMicronaut4OrGreater}}
 }
-{{#micronaut_serde_jackson}}
+{{#micronaut_serde_jackson}}{{^isMicronaut4OrGreater}}
```

Comment:
non shure about this, our micornaut 4 projects should not load micronaut-jackson-databind

Reply:
Non c'est bon, on devrait pas avoir besoin de ce bloc là pour MN 4.